src/utils/notifications.ts

```diff
@@ -390,6 +394,50 @@ export const sendFundsClaimedNotifications = async ({
   );
 };
 
+export const sendColonyVersionAddedNotifications = async (
+  newVersion: number,
+): Promise<void> => {
+  const allColonies = await getAllColoniesWithRootPermissionHolders();
+
+  await Promise.all(
+    allColonies.map(async (colony) => {
+      const recipients: Recipient[] = [];
+
+      if (!colony.roles?.items) {
+        return;
+      }
+
+      for (const roleItem of colony.roles.items) {
+        if (
+          !!roleItem?.targetUser?.notificationsData &&
+          shouldSendNotificationToRecipient(
+            roleItem.targetUser.notificationsData,
+            colony.id,
+          )
+        ) {
+          recipients.push({
+            external_id: roleItem.targetUser.notificationsData.magicbellUserId,
+          });
+        }
+      }
+
+      if (recipients.length > 0) {
+        await sendNotification(
+          `New colony version: ${newVersion}`,
+          recipients,
+          {
+            notificationType: NotificationType.NewColonyVersion,
+            notificationCategory: NotificationCategory.Mention,
+            colonyAddress: colony.id,
+            creator: colony.id, // who's the creator of this??
```

Comment:
good question 😄

Reply:
🤯  is the creator field optional? Maybe it's about time that it is!

Reply:
Otherwise yeah just the colony id is fine 😆